components/brave_new_tab_ui/new_tab_takeover/mojom/BUILD.gn

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 The Brave Authors. All rights reserved.
```

Comment:
this is under brave_new_tab_ui in components, but not in browser https://github.com/brave/brave-core/pull/27645/files#diff-f72ebad5725bed17e104d5245827d8d7cfdee3b6b9f1c7f122af3555e6ddd48c